src/ui/DynamicHierarchicalFacet/DynamicHierarchicalFacet.ts

```diff
@@ -149,6 +150,16 @@ export class DynamicHierarchicalFacet extends Component implements IDynamicHiera
      */
     enableMoreLess: ComponentOptions.buildBooleanOption({ defaultValue: true, section: 'CommonOptions' }),
 
+    /**
+     * Whether to allow the end-user to search the facet values.
+     *
+     * By default, the following behavior applies:
+     *
+     * - Enabled when more facet values are available.
+     * - Disabled when all available facet values are already displayed.
+     */
+    enableFacetSearch: ComponentOptions.buildBooleanOption({ section: 'Filtering' }),
```

Comment:
This description is copied from [DynamicFacets'](https://github.com/coveo/search-ui/blob/master/src/ui/DynamicFacet/DynamicFacet.ts#L192).